manifests/components/04d_argocd-server-deployment.yaml

```diff
@@ -33,6 +33,20 @@ spec:
         volumeMounts:
         - mountPath: /shared
           name: static-files
+
+        readinessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 5
+          periodSeconds: 10
```

Comment:
We can be more aggressive for readiness than liveness since it only happens during server startup. Plus our server comes up very quickly. We should the same healthz endpoint to verify the pod can talk to k8s before claiming ready:
```
          httpGet:
            path: /healthz
            port: 8080
          initialDelaySeconds: 2
          periodSeconds: 1
          failureThreshold: 30
```